### DISCUSSION TITLE
Z height on toolchange

### DISCUSSION THREAD

**Discussion by Gotdecibels** [Enraged-Rabbit-Community/ERCF_v2#130] [Q&A]
I can not figure out what to change to stop the z axis from touching down to the build plate then rising back to printing height after a toolchange.  Using prusa slicer and wipe tower when a toolchange occurs first it goes to wipe tower does it's little retraction, moves to cutters and cuts filament, unloads, loads, THEN DROPS Z AXIS TO THE BUILD PLATE, returns to proper printing height, and continues back to wipe tower. While this technically works I'd rather not move all the way down then back up .

**Comment by norris108** [comment]
This isn't an ERCF issue, but rather a [Happy Hare](https://github.com/moggieuk/Happy-Hare) configuration issue
